src/PhoneInputAsset.php

```diff
@@ -18,6 +18,7 @@ class PhoneInputAsset extends AssetBundle
     public $js = [
         'build/js/utils.js',
         'build/js/intlTelInput.min.js',
+        'build/js/intlTelInput-jquery.min.js',
```

Comment:
@i-panov I believe since https://github.com/jackocnr/intl-tel-input made a "separate jQuery wrapped version" - we should replace `build/js/intlTelInput.min.js` with `build/js/intlTelInput-jquery.min.js`, right? I don't think we should be using both at once

Reply:
@Borales Perhaps you are right. I did not understand what js files are needed or not needed by this library. I just saw that when intlTelInput-jquery.min.js was connected everything worked. But since if you disable `intlTelInput.min.js`, nothing breaks (I checked), you can remove it. A new problem is in another - in one of the latest updates `bower/intl-tel-input` the file `intlTelInput-jquery.min.js` was missing. And now if I try to connect it, I get 404. You do not know what happened there? I see that it is in the github repository, but it is not pumped out with a composer ...